vic/drivers/shared_image/include/vic_mpi.h

```diff
@@ -27,8 +27,7 @@
 #ifndef VIC_MPI_H
 #define VIC_MPI_H
 
-#include <stddef.h> // used for offsetof() macro
```

Comment:
I'm not finding `stddef.h` included anywhere else, but `offsetof()` is still used in `vic/drivers/shared_image/src/vic_mpi_support.c`